### ISSUE TITLE
Border radius not getting applied when parent view has border radius.

### ISSUE THREAD

**Issue by aflahkp** [facebook/react-native#26627]
When a view with border radius is written inside a parent view which also have a border radius, Then the childs border radius, and backgroundColor are not getting rendered.

I was trying to make a Custom Radio Button with a holo circular view as a parent and a filled circle within the parent. The inner filled circle not getting rendered properly.

React Native version: 0.61.1

## Steps To Reproduce
1. Create a View with non zero borderRadius in its style prop
2. Add a child view inside the view which also have a borderRadius in style prop
3. The childs borderRadius and backgroundColor, not geting rendered untill you remove parents borderRadius
4. My View is as bellow

```
  <View style={{
        height: 24,
        width: 24,
        borderRadius: 12,
        borderWidth: 2,
        borderColor: '#00b34f',
        alignItems: 'center',
        justifyContent: 'center',
      }}>
              <View style={{
               height: 12,
               width: 12,
               borderRadius: 6,
               backgroundColor: '#00b34f',
              }} /> 
</View>
```

**Comment by andvm** [comment]
Add information, child view(with text) rendered, background color child view NOT rendered
**Steps To Reproduce**
`<View style={{width: 300, height: 300, backgroundColor: 'red', borderRadius: 6}}>
         <View style={{width: 200, height: 200, backgroundColor: 'green', borderRadius: 4}}>
                 <Text>TEST TEXT</Text>
         </View>
</View>`

**Comment by andvm** [comment]
Fixed on 0.61.2

**Comment by aflahkp** [comment]
No. Still have the issue after updating to 0.61.2
